webapp/app/components/beam-files/component.js

```diff
@@ -62,6 +65,11 @@ export default Component.extend({
   actions: {
     filterChange: function(newFilter) {
       this.set("fileFilter", newFilter);
+    },
+    sortBy: function(property, acsending) {
+      var method = (acsending) ? 'asc': 'desc';
```

Comment:
Better use `const` or `let`

https://medium.com/@pandeysoni/when-should-use-const-and-let-instead-of-var-in-javascript-ec2c3d7e5ca6